server/tests/unit/services/RunService.test.js

```diff
@@ -224,18 +224,421 @@ describe('RunService', () => {
                 );
                 expect(activeBrowserVersionToAtVersions.length).toBe(1);
 
-                // This test should not change the active tech only the run
-                // because I only change the test version and APG example
-                // This is a bugs
-                // After this another test should be writtent hat is like
-                // excalty the same but the active tech also changes.
-                //
-                // Ideally another test maybe were you kept one and add a new
-                // one...
                 await tech.reload();
                 expect(tech.active).toBe(true);
             });
         });
+        it('should deactivate tech pairs when there is existing tech pair data to deactivate', async () => {
+            await dbCleaner(async () => {
+                const browserVersionNumber = '1.2.3';
+                const atVersionNumber = '3.2.1';
+
+                const browserVersionNumber2 = '3.4.5';
+                const atVersionNumber2 = '5.4.3';
+
+                let testVersion = await db.TestVersion.findOne({
+                    where: {
+                        git_hash: process.env.IMPORT_ARIA_AT_TESTS_COMMIT_1
+                    }
+                });
+
+                const at = await db.At.findOne({
+                    where: { test_version_id: testVersion.id },
+                    include: [db.AtName]
+                });
+
+                const browser = await db.Browser.findOne();
+
+                // Create tech pair to deactivate
+                let atVersion = await db.AtVersion.create({
+                    at_name_id: at.AtName.id,
+                    version: atVersionNumber
+                });
+
+                let browserVersion = await db.BrowserVersion.create({
+                    browser_id: browser.id,
+                    version: browserVersionNumber
+                });
+
+                const runStatus = await db.RunStatus.findOne({
+                    where: { name: db.RunStatus.RAW }
+                });
+
+                let techPairDeactivate = await db.BrowserVersionToAtVersion.create(
+                    {
+                        at_version_id: atVersion.id,
+                        browser_version_id: browserVersion.id,
+                        active: true,
+                        run_status_id: runStatus.id
```

Comment:
This is not an attribute on the tech pair.

Reply:
Good catch! Thank you